main/acle.md

```diff
@@ -1072,6 +1083,31 @@ including the header:
 
 Including `<arm_sme.h>` also includes [`<arm_sve.h>`](#arm_sve.h).
 
+### Predefined feature macros and header files
+
+Testing a feature macro returns the compiler's availability for the feature via ACLE
+documented intrinsics and inline assembly. This property can be independent of
+where the test is performed, meaning a feature macro does not guarantee the
+validity of using the feature at the point it is tested. For example:
```

Comment:
```suggestion
Feature macros indicate whether that feature's intrinsics and inline assembly instructions
are available in normal unannotated functions. In some implementations, the macro definitions
are invariant throughout a translation unit, while in other implementations, the macro
definitions can vary with pragmas.

The macro definitions are otherwise independent of where the test is performed,
meaning that a feature macro does not guarantee that the feature can be used at the
point that it is tested. For example:
```